addons/im_livechat/static/src/js/im_livechat_channel_form_controller.js

```diff
@@ -0,0 +1,44 @@
+odoo.define('im_livechat.ImLivechatChannelFormController', function (require) {
+'use strict';
+
+const FormController = require('web.FormController');
+
+const ImLivechatChannelFormController = FormController.extend({
+    events: Object.assign({}, FormController.prototype.events, {
+        'click .o_im_livechat_channel_form_button_colors_reset_button': '_onClickLivechatButtonColorsResetButton',
+        'click .o_im_livechat_channel_form_chat_window_colors_reset_button': '_onClickLivechatChatWindowColorsResetButton',
+    }),
+    /**
+     * @private
+     * @param {Object} colorValues
+     */
+    async _updateColors(colorValues){
```

Comment:
- whitespace before `{`

Now that you have a private method, I would introduce category splitters:
```js
//-----------------------------
// Private
//-----------------------------
```

(don't forget empty line before and after them)